### DISCUSSION TITLE
How to Upgrade Keycloak Docker container?

### DISCUSSION THREAD

**Discussion by ayoubAnbara** [keycloak/keycloak#15737] [Q&A]
I want to upgrade Keycloak Docker container from version 19.0.1 to 20.0.1.
The [Upgrading Guide](https://www.keycloak.org/docs/latest/upgrading/index.html#intro) that exists in Keycloak documentation does not talk about Docker container.

**Comment by sschu** [comment]
If you don't have any customisations in the image, it is as simple as stopping the old container, creating a backup and starting the new container.

**Reply by ayoubAnbara** [reply]
The new container should connect to the old database?

**Reply by sschu** [reply]
Of course, you don't want to lose your data, do you?
